sdk/tables/data-tables/src/TableClient.ts

```diff
@@ -493,31 +481,29 @@ export class TableClient {
     tableName: string,
     options: InternalListTableEntitiesOptions = {}
   ): AsyncIterableIterator<ListEntitiesResponse<TableEntityResult<T>>> {
-    const { span, updatedOptions } = createSpan("TableClient-listEntitiesPage", options);
-
-    try {
-      let result = await this._listEntities<T>(tableName, updatedOptions);
+    let result = await tracingClient.withSpan(
```

Comment:
I am going to thoroughly test how this looks now to make sure all looks ok as far as parenting, etc.

Also added the continuation token as an attribute on a span

Reply:
![image](https://user-images.githubusercontent.com/753570/154309008-397dd4b1-7caa-4067-9946-6daca64a0347.png)

Reply:
This is what it ends up looking like - which I think is what we landed on. With this in place I am thinking we can close #5607 - I have a separate issue tracking adding these as guidelines